tomcat/src/main/java/servlet/http/HttpMethod.java

```diff
@@ -0,0 +1,17 @@
+package servlet.http;
+
+import java.util.Arrays;
+
+public enum HttpMethod {
+
+    GET, POST,
+    ;
+
+    public static HttpMethod from(String method) {
+        return Arrays.stream(values())
+                .filter(httpMethod -> httpMethod.name().equalsIgnoreCase(method))
```

Comment:
[HTTP 표준](https://datatracker.ietf.org/doc/html/rfc7231#section-4.1)에 따라 http method가 대문자로 입력된 경우에만 유효하게 처리해주는 것은 어떨까요?

> The method token is case-sensitive because it might be used as a gateway to object-based systems with case-sensitive method names.